Plugins/Json/include/Acts/Plugins/Json/GeometryHierarchyMapJsonConverter.hpp

```diff
@@ -104,16 +104,17 @@ class GeometryHierarchyMapJsonConverter {
 // implementations
 
 template <typename value_t>
-nlohmann::json GeometryHierarchyMapJsonConverter<value_t>::toJson(
+nlohmann::ordered_json GeometryHierarchyMapJsonConverter<value_t>::toJson(
     const Container& container) const {
   // encode header
-  nlohmann::json encoded = nlohmann::json::object();
+  nlohmann::ordered_json encoded = nlohmann::json::object();
+  ;
   encoded[kHeaderKey] = {
       {"format-version", kFormatVersion},
       {"value-identifier", m_valueIdentifier},
   };
   // encode entries
-  nlohmann::json entries = nlohmann::json::array();
+  nlohmann::ordered_json entries = nlohmann::json::array();
```

Comment:
See above, but a JSON array is always ordered anyways.

Reply:
In that case if you use `nlohmann::json` you cannot use `.push_back` anymore. `json.push_back(ordered_json)` does not compile.